docker-compose.yml

```diff
@@ -5,13 +5,16 @@ services:
     build: .
     depends_on:
       - api
+      - redis
     ports:
       -  "5100:5100"
   api:
-    build: https://github.com/youpin-city/youpin-api.git
+    build: ../youpin-api
```

Comment:
Does this work with the url? I think in order to run `api`, we need to set up several things such as GCS credentials.